packages/react-dev-utils/getProcessForPort.js

```diff
@@ -58,7 +58,7 @@ function getProcessCommand(processId, processDirectory) {
 
 function getDirectoryOfProcessById(processId) {
   return execSync(
-    'lsof -p ' + processId + ' | awk \'$4=="cwd" {print $9}\'',
+    'lsof -p ' + processId + ' | awk \'$4=="cwd" {for (i=9; i<=NF; i++) printf "%s ", $i}\'',
```

Comment:
(Unfortunately, a `for` loop appears to be the most efficient way to do this; the alternative is to set fields 1-8 to blank strings, but that'd end up less clear as to the intent.)